.travis.yml

```diff
@@ -1,5 +1,13 @@
 language: c
+sudo: required
```

Comment:
That would be the start up time for `sudo` image.

Unfortunately, docker [requires `sudo` to build and run](https://docs.travis-ci.com/user/docker/). They should have add the test user to docker's system group so. They didn't.